lake/modules/onyx_pe.py

```diff
@@ -412,13 +397,26 @@ def get_bitstream(self, config_kwargs):
             10: 0b011,
             11: 0b011,
         }
-        sparse_num_inputs = op_sparse_num_inputs_mapping[op]
-        if config_kwargs["rb_const"] is not None:
-            # the b operand is a constant
-            # support constant operand for and and fp_mul for now
-            assert op == 5 or op == 6
-            # only accepting one input from port a【
-            sparse_num_inputs = 0b001
+        sparse_num_inputs = 0b000
+        if 'use_dense' in config_kwargs and config_kwargs['use_dense'] is True:
+            override_dense = True
+            config += [("dense_mode", 1)]
+        elif 'pe_connected_to_reduce' in config_kwargs and config_kwargs['pe_connected_to_reduce'] is True:
+            # If this flag is set, we want to connect the pe to reduce
```

Comment:
> # If this flag is set, we want to connect the pe to reduce

Thanks for adding comments! But this comment needs a little more thought maybe.
"to reduce"...reduce what? code size? by...? bypassing the fifos?

Reply:
reducer is a sparse primitive, @bobcheng15  can include that in the comments

Reply:
@steveri sorry for the confusion. Like Kalhan says, by setting this `pe_connected_to_reduce` flag, the ALU (or onyx_pe) is connected to the reduce sparse primitive. I will revise the comment to make it clear.